### DISCUSSION TITLE
Use two separate buttons for 'Create pull request' and 'Create draft pull request' instead of a dropdown

### DISCUSSION THREAD

**Discussion by brianlove** [community/community#63829] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

When creating a pull request, I find the current interface for selecting whether to create a PR that's ready for review versus a draft PR to be fairly clunky - there's a button, but also a dropdown, and selecting the other option from the dropdown changes the mode of the button instead of actually submitting something.  This creates extra steps when submitting a PR and the similar wording of the various options (and same visual style) requires careful attention to which mode is currently active.

<img width="442" alt="Screenshot 2023-08-15 at 11 04 07 AM" src="https://github.com/community/community/assets/22353962/772774ea-3a6f-488f-946d-b50dc2f5d5e9">

Instead, I would like to suggest that instead of the dropdown, separate buttons for 'Create pull request' and 'Create draft pull request' be present in that location.  'Create pull request' would be the primary (green) button, and be on the right since it's generally the default action.  Meanwhile, 'Create draft pull request' would be a secondary (gray) button to the left.  I've created the following mockup:

<img width="470" alt="Screenshot 2023-08-15 at 11 26 20 AM" src="https://github.com/community/community/assets/22353962/f98ccd36-1b5d-420c-a9bc-6ea3b0930e90">

This would simplify the interface (fewer bits of hidden interface), reduce clicks (since there's no need to open the dropdown), and streamline the PR creation process since the different actions would be visually distinct.

**Comment by bdach** [comment]
Tangentially related, but I find the fact that the draft/non-draft setting in the dropdown is remembered for subsequent PRs to be an anti-feature. As someone that opens 99% non-draft and 1% draft PRs, a draft PR screws me up almost every time without fail later, because I don't usually _want_ my PRs to be drafts except for rare cases, but muscle memory makes me click the green button and I accidentally open drafts that way when I didn't mean to.

Splitting the button as proposed above would let get me my muscle memory back and not make this mistake.
